core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -1799,6 +1799,20 @@ class SparkContext(config: SparkConf) extends Logging {
         // For local paths with backslashes on Windows, URI throws an exception
         addJarFile(new File(path))
       } else {
+        /**
```

Comment:
Nit: you don't want scaladoc syntax here, and the comment doesn't add anything anyway